src/Equinor.Procosys.Library.Query/GetFunctionalRoles/PersonInFunctionalRole.cs

```diff
@@ -8,6 +8,8 @@ public class PersonInFunctionalRole
 
         public string LastName { get; set; }
 
+        public string UserName { get; set; }
```

Comment:
why added? Not used

Reply:
It is included in the persons under functional roles now, which we use in IPO. Am I missing something maybe?

Reply:
New:
![image](https://user-images.githubusercontent.com/56581095/106011522-d5acfe80-60ba-11eb-984d-ad71ddde8589.png)
Old:
![image](https://user-images.githubusercontent.com/56581095/106011587-e78ea180-60ba-11eb-842e-35e4192b475e.png)

Reply:
No. Fine